ietf/doc/tests_ballot.py

```diff
@@ -1231,8 +1231,10 @@ def _assertBallotMessage(self, q, balloter, expected):
         heading = q(f'div.h5[id$="_{slugify(balloter.plain_name())}"]')
         self.assertEqual(len(heading), 1)
         # <div.h5> is followed by a panel with the message of interest, so use next()
+        next = heading.next()
+        print(f"looking for {expected} in:\n{next}")
```

Comment:
Leftover debugging?

Reply:
Fixed in b430fa2583446581c2de942618b1efba5cb496fd.